test/functionalTest/cases/613_POST_v1_contextEntities/v1_contextEntities_without_entity.test

```diff
@@ -31,9 +31,9 @@ brokerStart CB
 
 #
 # 01. POST /v1/contextEntities without entityId in JSON payload and see it fail
-# 02. POST /v1/contextEntities/EID with entityId:id=EID in JSON payload and see it fail
+# 02. POST /v1/contextEntities/EID with entityId:id=EID2 in JSON payload and see it fail
```

Comment:
Chaning EID with EID2 is only a cosmetic change or it is due to some other reason?